src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -415,14 +419,28 @@ GlobalAppSettings& CascadiaSettings::GlobalSettings()
     return _globals;
 }
 
+// Function Description:
+// - Returns true if the user has installed PowerShell Core. This will check
+//   both %ProgramFiles% and %ProgramFiles(x86)%, and will return true if
+//   powershell core was installed in either location.
+// Arguments:
+// - A ref of a path that receives the result of PowerShell Core pwsh.exe full path.
+// Return Value:
+// - true or false.
+bool CascadiaSettings::_IsPowerShellCoreInstalled(std::filesystem::path& cmdline)
+{
+    return _CheckPowerShellCoreInstalledInPath(L"%ProgramFiles%", cmdline) ? true :

```

Comment:
This is weird. Why isn't this just an || ?

Reply:
Because I wrote that code before the coffee kicked in :P Yea I'll fix that